queries/c/indents.scm

```diff
@@ -9,6 +9,12 @@
   (expression_statement)
 ] @indent.begin
 
+(
+ (parenthesized_expression) @indent.align
+ (#set! indent.open_delimiter "(")
+ (#set! indent.close_delimiter ")")
+)
+
```

Comment:
Is this the norm for C-style languages?

Reply:
I feel like C has definitive camps and I'd hate to be picking one vs the other.

see
https://codingart.readthedocs.io/en/latest/c/Formatting.html

it's not uncommon to want the multiline conditional statement to be aligned within the parenthesis - which with sw=4,ts=4 and just ```if ( ``` probably doesn't matter but would for a cascaded if with ```else if (``` where the normal indent would just be 4.

Aligned doesn't one vs the other that it just say it'll align if you start on the same line...

Open to feedback on that.

Reply:
When you get into C stylistics, there's a lot of different opinions

https://softwareengineering.stackexchange.com/questions/3450/most-readable-way-to-format-long-if-conditions

But most of them have some form of alignment with the open paren.

Reply:
There's different styles/preferences for sure with C. I'll check out how this works in a few

Reply:
> 

Through the first look, this seems to be a question of
- Are they indenting it to indent + 4
- Or are they indenting it so it aligns with the opening parenthesis

Reply:
I'm going to remove it from this PR - it's something to think about, but it wasn't the point, so it should go separately.

Reply:
Just FYI, not only does this depend on user preference, but imo it depends on where the parenthesized expression is too. Maybe an easy way to have user-configurable options/styles would be nice